source/endpoints.go

```diff
@@ -81,6 +81,17 @@ func endpointsForHostname(hostname string, targets endpoint.Targets, ttl endpoin
 	return endpoints
 }
 
+func EndpointsForHostname(
```

Comment:
Out of curiosity, I'm probably missing something here, but why exporting `endpointsForHostname` like this ? instead of just renaming it with a capitalized E in your editor ?

Reply:
I could do this, but is going to be too many changes in this PR.

I'll open a follow-up with the rename